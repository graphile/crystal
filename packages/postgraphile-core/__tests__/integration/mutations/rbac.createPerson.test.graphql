#> rbac: true
#> options: { ignoreRBAC: false }

mutation {
  createPerson(input:{
    person:{
      name:"Jane Doe"
      aliases: ["Jay Doe", "JD"]
      about: "Unknown"
      email: "jane.doe@example.com"
      site: {
        url: "http://example.com"
      }
    }
  }) {
    person{
      id
      name
      aliases
      about
      email
      site {
        url
      }
      #createdAt # Can't include - it changes
    }
  }
}
