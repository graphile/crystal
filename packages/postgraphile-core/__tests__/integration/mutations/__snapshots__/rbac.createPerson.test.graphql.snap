// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "createPerson": Object {
      "person": Object {
        "about": "Unknown",
        "aliases": Array [
          "Jay Doe",
          "JD",
        ],
        "email": "jane.doe@example.com",
        "id": 10,
        "name": "Jane Doe",
        "site": Object {
          "url": "http://example.com",
        },
      },
    },
  },
}
`;
